src/Page/Index.elm

```diff
@@ -41,3 +65,42 @@ view model =
 viewTextColumn : (Key -> String) -> List Key -> List (Html msg)
 viewTextColumn t paragraphs =
     List.map (\para -> p [ css [ pageColumnBlockStyle ] ] [ text (t para) ]) paragraphs
+
+
+guideLinkStyle : Style
+guideLinkStyle =
+    batch
+        [ textDecoration none
+        ]
+
+
+guideLinkTitleStyle : Style
+guideLinkTitleStyle =
+    batch
+        [ display inlineBlock
+        , fontSize (rem 1)
+        , fontFamilies [ "Rubik", "sans serif" ]
+        , fontWeight bold
```

Comment:
This isn't taking effect, looking at the font import in `index.html` I don't think we're importing anything but the `300` weight.

Reply:
Yes that would need a new font import